### PR TITLE
added missing route for authenticated() action.

### PR DESCRIPTION
**Pull request by thinkingmedia**

### CODE REVIEW

File: config/routes.php [hunk 1]
```diff
@@ -15,4 +15,8 @@
         '/endpoint',
         ['controller' => 'HybridAuth', 'action' => 'endpoint']
     );
+	$routes->connect(
```
Comment (ADmad) [inline config/routes.php]: Indentation is off.

File: src/Template/Element/login.ctp [hunk 2]
```diff
@@ -12,5 +12,4 @@ echo $this->Form->input(
 );
 
 echo $this->Form->submit('Login');
-echo $this->Form->end();
-?>
\ No newline at end of file
+echo $this->Form->end();
```
Comment (ADmad) [inline src/Template/Element/login.ctp]: As I commented on you previous PR i prefer keeping ?> in template files.

File: config/routes.php (hunk 1 above)
Comment (thinkingmedia) [inline config/routes.php]: my bad.

File: src/Template/Element/login.ctp (hunk 2 above)
Comment (thinkingmedia) [inline src/Template/Element/login.ctp]: sorry, it sneaked back in.

**Pull request closed without merging**